site/en/codelabs/openthread-simulation/index.lab.md

```diff
@@ -92,6 +92,9 @@ $ docker run --name codelab_otsim_ctnr -it --rm \
    --cap-add=net_admin openthread/environment bash
 ```
 
+The --rm option deletes the container when you exit the container. Do not use this option if you do not want the 
```

Comment:
Use code font for the --rm text (surround it with backticks)

`--rm`

Reply:
done, thanks, Jeff